src/auth/OidcCallback.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useTranslate, useNotify } from 'react-admin';
+import { CallbackComponent } from 'redux-oidc';
+import { User } from 'oidc-client';
+import { RouteChildrenProps } from 'react-router';
+
+import userManager from './userManager';
+import { fetchApiToken } from './api';
+
+function OidcCallback(props: RouteChildrenProps) {
+  const translate = useTranslate();
+  const notify = useNotify();
+
+  const onSuccess = (user: User) => {
+    fetchApiToken(user.access_token)
+      .then(apiToken => {
+        localStorage.setItem('apiToken', apiToken);
+        props.history.push('/');
+      })
+      .catch(error => {
+        notify(translate('ra.message.error'), 'warning');
+        // TODO Add Sentry
```

Comment:
would be best to delete token here when error ?
Or just doesnt need to care because this is admin UI ?

Reply:
You mean we should delete the access token here? TBH I haven't really examined possible error situations yet, so not sure what is the best reaction here.